mass/core/analysis_algorithms.py

```diff
@@ -279,29 +279,6 @@ def entropy(param, indicator, uncorrected, smoother):
 
 @njit
 def python_nearest_arrivals(reference_times, other_times):
-    """Identical to nearest_arrivals(...)."""
-    nearest_after_index = np.searchsorted(other_times, reference_times)
-    # because both sets of arrival times should be sorted, there are faster algorithms than searchsorted
-    # for example: https://github.com/kwgoodman/bottleneck/issues/47
-    # we could use one if performance becomes an issue
-    last_index = np.searchsorted(nearest_after_index, other_times.size, side="left")
-    first_index = np.searchsorted(nearest_after_index, 1)
-
-    nearest_before_index = np.copy(nearest_after_index)
-    nearest_before_index[:first_index] = 1
-    nearest_before_index -= 1
-    before_times = reference_times - other_times[nearest_before_index]
-    before_times[:first_index] = np.inf
-
-    nearest_after_index[last_index:] = other_times.size - 1
-    after_times = other_times[nearest_after_index] - reference_times
-    after_times[last_index:] = np.inf
-
-    return before_times, after_times
-
-
-@njit
-def nearest_arrivals(pulse_timestamps, external_trigger_timestamps):
```

Comment:
Why delete this code? It's an implementation of the faster and more efficient algorithm, seems useful to keep around if we ever want to switch.

Reply:
It didn't run. I guess that's my answer.

Reply:
I'm also concerned that the git diff made it confusing, because I adapted one function's docstring and the other's code. I will look at the two more carefully after I sleep and consider this objection.